### DISCUSSION TITLE
JSON Schema 2024 Budget Proposal

### DISCUSSION THREAD

**Discussion by benjagm** [json-schema-org/community#692] [Vision and direction]
Dear Community,

In our continuous effort to maintain transparency and foster discussion within the community, we are sharing the proposed budget for 2024. This overview reflects the financial planning work we are doing, identifying the operating costs of the project as well as cost estimation for new initiatives we're launching this year.

Below is a detailed breakdown of the budget items, along with their estimated costs. We're also highlighting new projects and initiatives planned for 2024, aiming to maximize the project impact and better server our users.

### 2024 Budget Overview

**Operating Costs**
| Item | Description | Estimated Cost |
| ---- | ----------- | -------------- |
Cloudflare | Website hosting services | 300
Plausible | GRP compliance analytics | 1000
Mailfence | Mail service | 42

**New initiatives:**

**1.- Newsletter**
| Item | Description | Estimated Cost |
| ---- | ----------- | -------------- |
Mailchimp  | Newsletter provider | 120

**2.- Swag kits + Online Shop to support Community Programs**
| Item | Description | Estimated Cost |
| ---- | ----------- | -------------- |
Online Shop (Development) | 1 time payment to setup the site | 398
Online Shop Hosting | Yearly costs of the site | 1200
Swag | Produce swag kits for Ambassadors, Implementers, Core Team and contributor of the month | 4000
Swag Storage | Yearly costs of store the produced swag | 1000

**3.- JSON Schema Conf**
| Item | Description | Estimated Cost |
| ---- | ----------- | -------------- |
Swag | Swag to give attendees | 2000
Booth Setup | Roll-up, banners and posters | 600

**4.- Hire a Community Manager**
| Item | Description | Estimated Cost |
| ---- | ----------- | -------------- |
Community Manager | In case of deciding hiring a new Community Manager | 12000

**Totals**
| Item  | Estimated Cost |
| ----- | -------------- |
| Operating costs | 1342 |
| Newsletter service | 120 |
| Swag kits + Online Shop to support Community Programs | 6598 |
| JSON Schema Conf | 2600 |
| Community Manager | 12000 | 
| Total | 22660 €|

**Total Estimated Budget for 2024: 22660 €**

**New Initiatives Highlights:**

- **Initiative 1**: [Newsletter]
- **Initiative 2**: [Swag kits + Online Shop to support Community Programs]
- **Initiative 3**: [JSON Schema Conf]
- **Initiative 4**: [Community Manager]

We believe these investments are essential for the sustained growth and improvement of the JSON Schema project. Each item has been carefully considered to ensure we're effectively utilizing our resources to benefit the community and further our mission.

**Your Feedback Matters:**

We welcome your thoughts, feedback, and any suggestions you may have regarding this proposed budget. Our goal is to ensure the JSON Schema project continues to thrive and meet the needs of our growing community.

If you have questions, ideas, or would like to contribute to any of the initiatives mentioned above, please leave a comment!

> Note: This budget is subject to change as we are still working on some costs estimations.

**Comment by jdesrosiers** [comment]
> 4.- Hire a Community Manager

I don't think this is something we can do from a legal standpoint. JSON Schema isn't a company, so it can't have employees. I think we're dependent on outside companies like Postman for this sort of thing.

**Reply by benjagm** [reply]
We can pay monthly through open collective like AsyncAPI is doing with Thulie, the Community Manager. It works like a grant.

**Reply by jdesrosiers** [reply]
Ok. As long as you know what you're doing. I certainly don't in these matters.

**Reply by gregsdennis** [reply]
We really need to consider things here. There are long-standing contributors _to the spec_ who haven't been compensated at all, and now we're looking at bringing in someone new and paying them?  That doesn't seem right to me.

**Comment by egekorkan** [comment]
A bit on point 4, after reading the spec lifecycle: IETF works differently but other standardization organizations work in a way that specification contributors are not compensated. Quite the opposite, they or the organization they are affiliated with, pay to contribute. This payment goes to costs like those mentioned above but also to staff, which can include a community manager, marketing staff, administrative, CEO, etc. The staff typically do not contribute to the specification text directly. So there is a rather clear separation of responsibilities. 

To be clear, I am not advocating for this approach or advocating against it. I just wanted to highlight one way such things are done.
